extensions/aria_extension_tosca/simple_v1_0/modeling/functions.py

```diff
@@ -69,7 +69,7 @@ def __evaluate__(self, container_holder):
             e, final = evaluate(e, final, container_holder)
             if e is not None:
                 value.write(unicode(e))
-        value = value.getvalue()
+        value = value.getvalue() or u''
```

Comment:
We default to Unicode everywhere possible in values -- this avoids thorny problems later down the road with values.